test/src/specs/relay/transaction_relay_low_fee_rate.rs

```diff
@@ -22,16 +22,12 @@ impl Spec for TransactionRelayLowFeeRate {
 
         info!("Generate new transaction on node1");
         node1.generate_blocks((DEFAULT_TX_PROPOSAL_WINDOW.1 + 2) as usize);
-        let hash = node1.generate_transaction();
+        let tx = node1.new_transaction_spend_tip_cellbase();
+        node1.submit_transaction(&tx);
+        let hash = tx.hash();
         // confirm tx
         node1.generate_blocks(20);
-        let ret = wait_until(10, || {
-            node1
-                .rpc_client()
-                .get_transaction(hash.clone())
-                .map(|tx_result| tx_result.tx_status.status == Status::Committed)
-                .unwrap_or(false)
-        });
+        let ret = wait_until(10, || is_transaction_committed(node1, &tx));
```

Comment:
There seems doesn't need `wait_until`?

Reply:
Yes, I think, `wait_until` is unnecessary here. However, I would rather leave it the same, since I only want to replace it with the new checking suites, but not improve the test codes.